pt/lessons/advanced/umbrella-projects.md

```diff
@@ -10,13 +10,13 @@ Determinados momentos um projeto pode ficar enorme, realmente enorme. A ferramen
 
 ## Sumário
 
-- [Introdução](#introducao)
+- [Introdução](#introdução)
```

Comment:
You should not use accentuation for anchor links. Use `#introducao` and the link on title. 

You can check the `pipe-operator.md` in pt-BR